src/utils/__tests__/csrfUtils.test.ts

```diff
@@ -16,7 +16,7 @@ jest.mock("crypto", () => {
   };
 });
 
-describe("validateCsrfToken", () => {
+describe.skip("validateCsrfToken", () => {
```

Comment:
should we remove this test?

Reply:
Yes, I was debating keeping it or not but I'll just remove.